src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -483,10 +484,27 @@ private void UpdateAdditionalFiles(Project project, IList<string> additionalFile
 
             foreach (var file in additionalFiles)
             {
-                 if (File.Exists(file))
-                 {
+                if (File.Exists(file))
+                {
                     _workspace.AddAdditionalDocument(project.Id, file);
-                 }
+                }
+            }
+        }
+
+        private void UpdateAnalyzerConfigFiles(Project project, IList<string> analyzerConfigFiles)
```

Comment:
should this code be conditioned on editorconfig being enabled in OmniSharp?
otherwise, if you disable it, it would be respected on formatting, but the analyzer options would still apply